dev-libs/libgnt/libgnt-2.14.1.recipe

```diff
@@ -0,0 +1,87 @@
+SUMMARY="Text-mode UI toolkit"
+DESCRIPTION="libgnt is a GLib-based ncurses toolkit for creating text-mode \
+graphical user interfaces quickly and easily."
+HOMEPAGE="https://pidgin.im/"
+COPYRIGHT="2006-2021 libgnt contributors"
+LICENSE="GNU GPL v2"
+REVISION=1
+SOURCE_URI="http://downloads.sourceforge.net/project/pidgin/libgnt/$portVersion/libgnt-$portVersion.tar.xz"
+CHECKSUM_SHA256="5ec3e68e18f956e9998d79088b299fa3bca689bcc95c86001bc5da17c1eb4bd8"
+SOURCE_DIR="libgnt-$portVersion"
+PATCHES="libgnt-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+versionCompat="$portVersion compat >= ${portVersion%%.*}"
+
+PROVIDES="
+	libgnt$secondaryArchSuffix = $portVersion
+	lib:libgnt$secondaryArchSuffix = $versionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libncursesw$secondaryArchSuffix
+	lib:libpanelw$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:libgobject_2.0$secondaryArchSuffix
+	lib:libgmodule_2.0$secondaryArchSuffix
+	lib:libgthread_2.0$secondaryArchSuffix
+	"
+PROVIDES_devel="
+	libgnt$secondaryArchSuffix = $portVersion
+	devel:libgnt$secondaryArchSuffix = $versionCompat
+	"
+REQUIRES_devel="
+	haiku${secondaryArchSuffix}_devel
+	libgnt$secondaryArchSuffix == $portVersion base
+	"
+if [ -z "$secondaryArchSuffix" ]; then
+	ARCHITECTURES_doc="any"
+	PROVIDES_doc="
+		libgnt_doc = $portVersion
+		"
+	REQUIRES_doc=""
+fi
```

Comment:
As we don't build for primary architecture on 32bit this can be removed here (and later on in INSTALL)
```diff
-if [ -z "$secondaryArchSuffix" ]; then
-	ARCHITECTURES_doc="any"
-	PROVIDES_doc="
-		libgnt_doc = $portVersion
-		"
-	REQUIRES_doc=""
-fi
```